installer/datafiles/base_omsagent.data

```diff
@@ -32,7 +32,7 @@ MAINTAINER:              'Microsoft Corporation'
 /etc/opt/microsoft/omsagent/sysconf/omsagent.d/collectd.conf;           installer/conf/omsagent.d/collectd.conf;               644; root; root
 /etc/opt/microsoft/omsagent/sysconf/omsagent.d/statsd.conf;             installer/conf/omsagent.d/statsd.conf;                 644; root; root
 /etc/opt/microsoft/omsagent/sysconf/omsagent.d/auditlog.conf;           installer/conf/omsagent.d/auditlog.conf;               644; root; root
-/etc/opt/microsoft/omsagent/sysconf/omsagent.d/mongostat.conf;          installer/conf/omsagent.d/mongostat.conf;              644; root; root
```

Comment:
Yes, I saw it too . Sorry hit the request button too quick . Will have it fixed in the next iteration. Thanks.